ocaml/src/gobjfs/test.ml

```diff
@@ -0,0 +1,119 @@
+open Gobjfs
```

Comment:
This file `ocaml/src/gobjfs/test.ml` looks like it's a duplicate of `ocaml/src/gobjfs/gobjfs_test.ml` (except for the missing license header), you probably want to remove it.